enhancements/installer/vsphere-ipi-zonal.md

```diff
@@ -180,6 +219,57 @@ type VSpherePlatformTopology struct {
 	// +kubebuilder:validation:Pattern=`^/.*?/vm/.*?`
 	// +optional
 	Folder string `json:"folder,omitempty"`
+
+	// template is the full inventory path of the virtual machine or template
```

Comment:
If hosts are being used as zones, how will that information reflected in `Infrastructure` type?

Reply:
The infrastructure spec will have the name of the vm-host group of type hosts

We will not know the ESXi hosts directly that are in each zone. I don't think we should either since stretched clusters presumably should already have this defined.

Reply:
But then, it sounds like customer must create a vm-host group for each host? I must be misunderstanding something. The CSI driver doesn't support hostgroup as a failure-domain fwiw.

Reply:
updated with https://github.com/openshift/enhancements/pull/1677/commits/8c3f02c7a334d02dc5949dbde2dd9907259554d1